content/includes/header.html

```diff
@@ -67,11 +67,7 @@
 <!-- We participate in the US government's analytics program. See the data at analytics.usa.gov. -->
 <script async type="text/javascript" src="https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js?agency=VA" id="_fed_an_ua_tag"></script>
 
-<script>
-//<![CDATA[
-window.webpackManifest = 'CHUNK_MANIFEST_PLACEHOLDER';
-//]]>
-</script>
+<script src="/generated/chunk-manifest.js"/></script>
```

Comment:
slight issue here - since the file doesn't exist in development we get a 404

Reply:
would it make sense to do this async or no?